pymongo/daemon.py

```diff
@@ -54,12 +54,16 @@ def _silence_resource_warning(popen):
 
     def _spawn_daemon(args):
         """Spawn a daemon process (Windows)."""
-        with open(os.devnull, 'r+b') as devnull:
-            popen = subprocess.Popen(
-                args,
-                creationflags=_DETACHED_PROCESS,
-                stdin=devnull, stderr=devnull, stdout=devnull)
-            _silence_resource_warning(popen)
+        try:
+            with open(os.devnull, 'r+b') as devnull:
+                popen = subprocess.Popen(
+                    args,
+                    creationflags=_DETACHED_PROCESS,
+                    stdin=devnull, stderr=devnull, stdout=devnull)
+                _silence_resource_warning(popen)
+        except FileNotFoundError as exc:
+            raise Exception(
```

Comment:
Instead of Exception can we raise `PyMongoError`?

Reply:
I think you forgot about this one.

Reply:
yep just got to this